src/os/annotation/abstractannotationctrl.js

```diff
@@ -73,7 +75,8 @@ os.annotation.UI_TEMPLATE =
           '</button>' +
         '</div>' +
       '</div>' +
-    '</div>';
+    '</div>' +
+  '</div>';
```

Comment:
This looks strange. Where did the extra opening `<div>` come from?

Reply:
When I was working on this I pulled the string out to it's own file. While it was separated, with syntax highlighting I noticed the missing closing tag. I didn't do a blame to see how long it's been like this.